notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -8,12 +8,12 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #91.                                         distance[i][j - 1] + 1,
</pre><p>logically, this part should be moved after quantization, in section where you check models accuracy, also some comments about what is it should be return back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1059/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>